static/js/components/PersonalTab.js

```diff
@@ -57,13 +57,14 @@ class PersonalTab extends ProfileTab {
       )}<br />
       <h4>Where do you live?</h4>
       {this.boundTextField(['city'], 'City')}<br />
-      {this.boundTextField(['state_or_territory'],'State or Territory')}<br />
+      {this.boundStateSelectField(['state_or_territory'], ['country'], 'State or Territory')}<br />
```

Comment:
I think it might make sense to move the state select after the country select, otherwise if you don't have a country selected you see this:

![states_and_territories](https://cloud.githubusercontent.com/assets/6207644/14990883/cc215eb6-112c-11e6-8922-ef5775aafa87.png)